client/config/default.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Why was this file removed? This can't be merged

Reply:
My bad. Accidentally committed changes that shouldn't have been in there. Didn't mean to delete them from the branch entirely :/